src/parsers/urdf/model.hxx

```diff
@@ -365,7 +365,10 @@ inline void parseTree (::urdf::LinkConstPtr link, Model & model, const SE3 & pla
         {
           model.appendBodyToJoint(parent_joint_id, jointPlacement, Y, link->name); //Modify the parent inertia in the model
         }
-        
+        else
+        {
+          model.addFrame(link->name, parent_joint_id, nextPlacementOffset, BODY);
```

Comment:
It is for the link that is attached to the fixed joint in case of no inertia.
In the if statement in the method appendBodyToJoint there is a call to addFrame. For links with no inertial information, the parser did not keep track of them ( was the case for the r_sole, l_sole in our romeo model)